src/explore-education-statistics-common/src/components/CollapsibleList.tsx

```diff
@@ -1,78 +1,72 @@
 import ButtonText from '@common/components/ButtonText';
 import useToggle from '@common/hooks/useToggle';
 import classNames from 'classnames';
-import React, {
-  Children,
-  cloneElement,
-  isValidElement,
-  ReactNode,
-} from 'react';
-import styles from './CollapsibleList.module.scss';
+import React, { Children, createElement, ReactNode } from 'react';
 
 interface Props {
   children: ReactNode;
   collapseAfter?: number;
+  id: string;
+  itemName?: string;
+  itemNamePlural?: string;
   isCollapsed?: boolean;
   listStyle?: 'none' | 'number' | 'bullet';
-  testId?: string;
 }
 
 const CollapsibleList = ({
   children,
   collapseAfter = 5,
+  id,
+  itemName = 'item',
+  itemNamePlural: initialItemNamePlural,
   isCollapsed = true,
   listStyle = 'none',
-  testId,
 }: Props) => {
   const [collapsed, toggleCollapsed] = useToggle(isCollapsed);
 
-  const listClasses = classNames('govuk-list', styles.listContainer, {
+  const itemNamePlural = initialItemNamePlural ?? `${itemName}s`;
+
+  const listItems = Children.toArray(children);
+
+  const renderedListItems = collapsed
+    ? listItems.splice(0, collapseAfter)
```

Comment:
Don't use `splice`, this is mutative and affects the original `listItems` variable. Use `slice` instead.

Reply:
Good point, switched to slice now.